pkgs/by-name/fr/freeplane/package.nix

```diff
@@ -5,22 +5,23 @@
   makeBinaryWrapper,
   makeDesktopItem,
   jdk17,
-  gradle_7,
+  gradle_8,
   which,
 }:
 
 let
   pname = "freeplane";
-  version = "1.11.14";
+  version = "1.12.7";
 
   jdk = jdk17;
-  gradle = gradle_7;
+  gradle = gradle_8;
 
   src = fetchFromGitHub {
     owner = "freeplane";
     repo = "freeplane";
-    rev = "release-${version}";
-    hash = "sha256-zEQjB57iiKVQnH8VtynpEGKNAa2e+WpqnGt6fnv5Rjs=";
+    # 1.12.7 was not tagged
```

Comment:
1.12.8 was https://github.com/freeplane/freeplane/releases/tag/release-1.12.8 lets use that